docs/Development.md

```diff
@@ -0,0 +1,175 @@
+<!-- TOC -->
+
+- [Have Questions](#have-questions)
+    - [Mailing Lists](#mailing-lists)
+    - [JIRA issues](#jira-issues)
+- [How to contribute](#how-to-contribute)
+    - [Becoming a committer](#becoming-a-committer)
+        - [Contributing by Helping Other Users](#contributing-by-helping-other-users)
+        - [Contributing by Testing Releases](#contributing-by-testing-releases)
+        - [Contributing by Reviewing Changes](#contributing-by-reviewing-changes)
+        - [Contributing by Documentation Changes](#contributing-by-documentation-changes)
+        - [Contributing Bug Reports](#contributing-bug-reports)
+        - [Contributing Code Changes](#contributing-code-changes)
+            - [Cloning source code](#cloning-source-code)
+            - [JIRA](#jira)
+            - [Pull Request](#pull-request)
+            - [The Review Process](#the-review-process)
+            - [Closing Your Pull Request / JIRA](#closing-your-pull-request--jira)
+            - [Code Style](#Code-Style)
+
+<!-- /TOC -->
+
+# Have Questions
+
+## Mailing Lists
+
+It is recommended to use our mailing lists to ask for help, debug issues or contribute to the project.
+
+* dev@iotdb.apache.org is for people who want to contribute codes to IoTDB or have usage questions for IoTDB.
+
+Some quick tips when using email:
+* For error logs or long code examples, please use GitHub gist and include only a few lines of the pertinent code / log within the email.
+* No jobs, sales, or solicitation is permitted on the Apache IoTDB mailing lists.
+
+PS. To subscribe our mail list, you can send an email to dev-subscribe@iotdb.incubator.apache.org and you will receive a "confirm subscribe to dev@iotdb.apache.org" email, follow the steps to confirm your subscribe.
+
+## JIRA issues
+
+The project tracks issues and new features on [JIRA issues](https://issues.apache.org/jira/projects/IOTDB/issues). You can create a new issue to report a bug, request a new feature or provide your custmon issue.
+
+# How to contribute
+
+## Becoming a committer
+
+To become a committer, you should first be active on our community so that most of our existing committers recognize you. Pushing codes and creating pull requests is just one of committer's rights. Moreover, it is committer's duty to help new uesrs on mail list, test new releases and improve documentation.
+
+### Contributing by Helping Other Users
+
+Since IoTDB always attracts new users, it would be great if you can help them by answering questions on the dev@iotdb.apache.org mail list. We regard it as a valuable contribution. Also, the more questions you answer, the more poeple know you. Popularity is one of the necessary conditions to be a committer.
```

Comment:
```suggestion
Since Apache IoTDB always attracts new users, it would be great if you can help them by answering questions on the dev@iotdb.apache.org mailing list. We regard it as a valuable contribution. Also, the more questions you answer, the more people know you. Popularity is one of the necessary conditions to be a committer.
```